### PR TITLE
docs: update the library name

### PR DESCRIPTION
**Pull request by droshev**
#### Please provide a link to the associated issue.
fixes: #1846 
fixes: #1610 
fixes: #1810 
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: NEW_COMPONENT.md [hunk 1]
```diff
@@ -51,17 +51,17 @@ Create an `exports` array in the poster module and add the poster component, lik
...
-The Fundamental NGX library uses 'fd' as the component and directive prefix. Open `poster.component.ts` and change the component's 'app' prefix to 'fd', like so:
+The Fundamental Library for Angular uses 'fd' as the component and directive prefix. Open `poster.component.ts` and change the component's 'app' prefix to 'fd', like so:
 
 ```TypeScript
   selector: 'fd-poster',
 ```
 
 ## Step5: Add poster module to list of imports (can be in FundamentalNgxCore)
 
-The documentation application is importing every component in the fundamental-ngx library module. Open `fundamental-ngx.module.ts` and add `import { PosterModule } from './poster/poster.module';` to the list of imports at the top of the file, then add `PosterModule` to the array of exports.
```
Comment (InnaAtanasova) [inline NEW_COMPONENT.md]: fundamental-ngx is a file name. Should not be renamed

File: NEW_COMPONENT.md [hunk 2]
```diff
@@ -51,17 +51,17 @@ Create an `exports` array in the poster module and add the poster component, lik
...
+The documentation application is importing every component in the Fundamental Library for Angular module. Open `Fundamental Library for Angular.module.ts` and add `import { PosterModule } from './poster/poster.module';` to the list of imports at the top of the file, then add `PosterModule` to the array of exports.
 
-We must also add `export * from './lib/poster/poster.module';` to the `fundamental-ngx/src/public_api.ts` file. The <fd-poster> component will be an exported member of the fundamental-ngx module as well as the poster module.
```
Comment (InnaAtanasova) [inline NEW_COMPONENT.md]: same here `fundamental-ngx module` is a file name

File: NEW_COMPONENT.md (hunk 1 above)
Comment (droshev) [reply NEW_COMPONENT.md]: done

File: NEW_COMPONENT.md (hunk 2 above)
Comment (droshev) [reply NEW_COMPONENT.md]: done

**Pull request merged**